src/main/java/org/truffleruby/interop/OutgoingForeignCallNode.java

```diff
@@ -23,22 +37,8 @@
 import com.oracle.truffle.api.interop.UnknownIdentifierException;
 import com.oracle.truffle.api.interop.UnsupportedMessageException;
 import com.oracle.truffle.api.interop.UnsupportedTypeException;
-import com.oracle.truffle.api.nodes.ExplodeLoop;
 import com.oracle.truffle.api.nodes.Node;
 import com.oracle.truffle.api.profiles.BranchProfile;
-import org.jcodings.specific.UTF8Encoding;
-import org.truffleruby.Log;
-import org.truffleruby.core.rope.CodeRange;
-import org.truffleruby.core.string.StringNodes;
-import org.truffleruby.language.RubyNode;
-import org.truffleruby.language.control.JavaException;
-import org.truffleruby.language.control.RaiseException;
-import org.truffleruby.language.dispatch.CallDispatchHeadNode;
-import org.truffleruby.language.dispatch.DispatchNode;
-import org.truffleruby.language.methods.ExceptionTranslatingNode;
-import org.truffleruby.language.methods.UnsupportedOperationBehavior;
-
-import java.util.Arrays;
```

Comment:
Did you configure your IDE to always organize/sort imports on save?
Unfortunately Eclipse and IntelliJ will fight on order, so it's better to disable auto ordering of imports, as it creates many conflicts (easy to solve but annoying and likely creating more conflicts).
It doesn't matter much for this specific PR.

Reply:
ok, I'll see if I can configure this per workspace. We use that behavior in Graal.js, but everybody is using Eclipse there ;-)